src/node.cc

```diff
@@ -589,7 +589,7 @@ Local<Value> ErrnoException(Isolate* isolate,
   }
   e = Exception::Error(cons);
 
-  Local<Object> obj = e->ToObject(env->isolate());
+  Local<Object> obj = e->ToObject(env->context()).ToLocalChecked();
```

Comment:
I think `Exception::Error` always returns an object, so `e.As<Object>()` should also work fine (and is a bit simpler/a bit more performant)

(This applies to a lot of other places in the PR too)

Reply:
Thank you for your review.

I see.
I didn't know that you could do it like that!